python/ray/autoscaler/vsphere/defaults.yaml

```diff
@@ -0,0 +1,120 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: default
+
+# The maximum number of workers nodes to launch in addition to the head
+# node.
+max_workers: 2
+
+# The autoscaler will scale up the cluster faster with higher upscaling speed.
+# E.g., if the task requires adding more nodes then autoscaler will gradually
+# scale up the cluster in chunks of upscaling_speed*currently_running_nodes.
+# This number should be > 0.
+upscaling_speed: 1.0
+
+# This executes all commands on all nodes in the docker container,
+# and opens all the necessary ports to support the Ray cluster.
+# Empty string means disabled.
+docker: {}
+
+# If a node is idle for this many minutes, it will be removed.
+idle_timeout_minutes: 5
+
+# Cloud-provider specific configuration.
+provider:
+    type: vsphere
+
+    vsphere_config:
+      credentials:
+        admin_user: <username>
+        admin_password: <password>
+        server: <server-ip>
+      datacenter: <datacenter>
+
+# How Ray will authenticate with newly launched nodes.
+auth:
+    ssh_user: ray
+# By default Ray creates a new private keypair, but you can also use your own.
+# If you do so, make sure to also set "KeyName" in the head and worker node
+# configurations below.
+#    ssh_private_key: /path/to/your/key.pem
+
+# Tell the autoscaler the allowed node types and the resources they provide.
+# The key is the name of the node type, which is just for debugging purposes.
+# The node config specifies the launch config and physical instance type.
+available_node_types:
+    ray.head.default:
+        # For example, {"CPU": 4, "Memory": 8192}
+        resources: {}
+        node_config:
+            # The resource pool where the head node should live
+            resource_pool: <resource_pool_name>
```

Comment:
I think fields from `defaults.yaml` get loaded into config files whenever the fields aren't provided by the user.  I'm assuming these fields `resource_pool`, `frozen_vm_name` and `datastore` are mandatory.  I'm worried that the string `<resource_pool_name>` will be loaded if the user forgets to provide it, and the resulting error message might be confusing, can you check this?  Is it better to comment out all the `<xxx>` fields from `defaults.yaml` altogether?

Reply:
Have removed the <xxx> fields and leave them empty or with a default value.